### DISCUSSION TITLE
How can I use a different profile for a specific provider?

### DISCUSSION THREAD

**Discussion by litoj** [ibhagwan/fzf-lua#1212] [Q&A] [answered]
I like `skim` - it's a lot faster than `fzf`, especially on larger file amounts. But I don't really like their fuzzy matching algorithm and also miss the enboldening of focused entry. I don't mind this for most providers, except `oldfiles`.

Is there a way I can have `skim` as my global profile, but use `fzf` for the `oldfiles` provider?

**Comment by ibhagwan** [comment, marked as answer]
You can use whatever you want on any provider, profiles are just a group of settings. Add `fzf_bin` (and fzf_opts if you want) to your setup under `oldfiles` or use it in the command directly:
```lua
-- fzf_bin also accepts fully qualified paths
:FzfLua oldfiles fzf_bin=fzf
```

**Reply by litoj** [reply]
Oh, I overlooked that option, thought it was just a comment to the option bellow. I will also look for some boldening options for skim and add another comment, if I manage to figure it out.

**Reply by ibhagwan** [reply]
It seems skim does ignore `bold` definition in the colors so idk if you can fix that, you can try the different algorithms though:
```lua
:FzfLua files fzf_opts.--algo=skim_v2
```

From `man sk`:
```
       --algo=TYPE
              Fuzzy matching algorithm (default: skim_v2)

              skim_v2 Almost always the one to choose
              skim_v1 The legacy algorithm
              clangd  the one used by clangd for keyword completion
```

**Reply by litoj** [reply]
Unfortunately none of them behave like fzf - I was looking for its matching by first characters of word -> first result for `sts` would be `syntax-tree-surfer`, instead of skim's `Tests`

**Reply by ibhagwan** [reply]
Unfortunately it does seem like skim has been abandoned, I wouldn’t recommend using it, you’re also missing out on UX improvements, for example try to stage/unstage with git_status with both skim and a newer version of fzf (there would be no flicker or process restart with fzf).
